miner/worker.go

```diff
@@ -174,6 +83,17 @@ type task struct {
 	createdAt time.Time
 }
 
+const (
+	signalNull int32 = iota
+	signalNewHead
+	signalResubmit
```

Comment:
Lets call these `commitInterruptNone`, `commitInterruptNewHead` and `commitInterruptResubmit`. I know they are a lot longer, but it's better to be a bit more explicit with global scoping, otherwise seeing a random `signalNull` in the code is hard on the reader.